<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14867" started-at="2020-02-08T15:31:40Z" finished-at="2020-02-08T15:31:55Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="14867" started-at="2020-02-08T15:31:40Z" finished-at="2020-02-08T15:31:55Z">
      <class name="Feb6.OrangeHRM">
        <test-method status="PASS" signature="SetUp()[pri:0, instance:Feb6.OrangeHRM@cb5822]" name="SetUp" is-config="true" duration-ms="13774" started-at="2020-02-08T21:01:40Z" finished-at="2020-02-08T21:01:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="SKIP" signature="VerifyLogin(java.lang.String, java.lang.String)[pri:0, instance:Feb6.OrangeHRM@cb5822]" name="VerifyLogin" duration-ms="0" started-at="2020-02-08T21:01:54Z" data-provider="datasupply" finished-at="2020-02-08T21:01:54Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void Feb6.OrangeHRM.VerifyLogin(java.lang.String,java.lang.String) throws java.lang.Throwable requires a @DataProvider named : datasupply]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void Feb6.OrangeHRM.VerifyLogin(java.lang.String,java.lang.String) throws java.lang.Throwable requires a @DataProvider named : datasupply
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:263)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:419)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:Feb6.OrangeHRM@cb5822]" name="tearDown" is-config="true" duration-ms="1020" started-at="2020-02-08T21:01:54Z" finished-at="2020-02-08T21:01:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Feb6.OrangeHRM -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
